mitmproxy/command.py

```diff
@@ -186,10 +196,15 @@ def parse_partial(
                 if parts[i] in self.commands:
                     params.extend(self.commands[parts[i]].paramtypes)
             elif params:
-                typ = params.pop(0)
-                if typ == mitmproxy.types.Cmd and params and params[0] == mitmproxy.types.Arg:
-                    if parts[i] in self.commands:
-                        params[:] = self.commands[parts[i]].paramtypes
+                if parts[i].strip() != '':
```

Comment:
`elif params and not parts[i].isspace():` :)

Reply:
Fixed